hack/gen-openapi-spec/main.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2019 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/go-openapi/spec"
+	tfjobspec "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1"
+	"k8s.io/kube-openapi/pkg/common"
+)
+
+// Generate OpenAPI spec definitions for KFService Resource
```

Comment:
KFService Resource -> TFJob

Reply:
Can we add a comment to show https://github.com/argoproj/argo/tree/master/hack/gen-openapi-spec ?

I think it will help us maintain the tool well.